apicast/bin/apicast

```diff
@@ -110,6 +110,7 @@ done
 
 main+=("daemon ${daemon};")
 main+=("worker_processes ${worker_processes};")
+main+=$(printenv | awk '$1 ~ /^(APICAST|THREESCALE)_/ {split($0,env,"="); print "env", env[1] ";"}')
```

Comment:
1. Hm, OK, I thought it was just a handy way to override NGINX default values. So, as it's not the case, I think it might complicate the transition to v2/v3 for the existing customers using self-managed deployment. Ideally, APIcast would work if you just drop the contents of the repo to the default nginx conf directory and restart the process.
But if we can't have it, I guess we'd need some kind of transition guide. @3scale/support what do you think?

2. My point is that I as a user want to have a complete list of the env vars that APIcast "understands", so I know what to export. If it's not in the code (configuration in this case) itself, it's OK, but we just need to make sure that the doc is always in sync.

3. OK, I was not aware of this requirement.